SU-SWS/Stanford-Drupal-Profile
# Enable stanford_page and stanford_date_formats by default

See https://asjira.stanford.edu/jira/browse/DRPL-479

To test:
1. Build a group or dept site with `make/group.make` or `make/dept.make`
2. Check out this branch of the profile in `profiles/stanford`
3. Install the `stanford` profile (`drush si stanford`)
4. Verify that `stanford_page` and `stanford_date_formats` are enabled.
